src/Hl7.Fhir.Base/Utility/CodedWithLocationException.cs

```diff
@@ -0,0 +1,72 @@
+﻿/* 
+ * Copyright (c) 2021, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://raw.githubusercontent.com/FirelyTeam/firely-net-sdk/master/LICENSE
+ */
+
+using Hl7.Fhir.Model;
+using System;
+
+#nullable enable
+
+namespace Hl7.Fhir.Utility
+{
+    /// <summary>
+    /// This is a class of Exceptions that is raised by the SDK and is coded with a unique code
+    /// to enable callers to identify this exception and react appropriately on the code.
+    /// </summary>
+    /// <remarks>Most modules of the SDK using this Exception will create specific subclasses
+    /// for this subclass, providing a list of codes used by that module.</remarks>
+    public class CodedWithLocationException : CodedException
+    {
+        public CodedWithLocationException(string errorCode, string message, OperationOutcome.IssueSeverity issueSeverity, OperationOutcome.IssueType issueType) : base(errorCode, message)
+        {
+            IssueSeverity = issueSeverity;
+            IssueType = issueType;
+        }
+
+        public CodedWithLocationException(string errorCode, string message, OperationOutcome.IssueSeverity issueSeverity, OperationOutcome.IssueType issueType, Exception? innerException) : base(errorCode, message, innerException)
+        {
+            IssueSeverity = issueSeverity;
+            IssueType = issueType;
+        }
+
+        /// <summary>
```

Comment:
I know it's a small thing, but can you put a dot (`.`) at the end of the sentences in the doccomments?

Reply:
Added in this file (will keep an eye on that from now)